malcontent.go

```diff
@@ -363,9 +371,24 @@ func main() {
 					switch {
 					case c.String("image") != "":
 						mc.OCI = true
-					case c.String("image") == "":
+					case c.String("image") == "" && !c.Bool("processes"):
```

Comment:
This condition was needed to prevent a no-op exit if no scan paths were provided to the command.